2364/2/parser.rb

```diff
@@ -0,0 +1,64 @@
+require 'russian_obscenity'
+require 'terminal-table'
+load 'participant.rb'
+load 'modules.rb'
+
+# Class parse for parsing data
+class Parser
+  PATH = File.expand_path('.').freeze
+  attr_reader :participants, :methods
+  def initialize(value, name)
+    @participants = []
+    @methods = create_methods
+    create_participants(Dir[PATH + '/rap-battles/*'])
```

Comment:
it's definetely not a good option to hard-code it this way, better to move it to constant outside and pass as param